lib/public/views/Runs/ActiveColumns/runDetectorsAsyncQcActiveColumns.js

```diff
@@ -29,7 +30,15 @@ import { QC_SUMMARY_COLORS } from '../../../components/qcFlags/qcSummaryColors.j
  */
 const runDetectorAsyncQualityDisplay = (runNumber, dplDetectorId, { dataPassId, simulationPassId }, qcSummary) => {
     if (!qcSummary?.[runNumber]?.[dplDetectorId]) { // If there is no QC flag assigned
-        return qcFlagCreationPanelLink({ dataPassId, simulationPassId, runNumber, dplDetectorId });
+        return qcFlagCreationPanelLink(
+            { dataPassId, simulationPassId, runNumber, dplDetectorId },
+            {
+                noPermissionContent: tooltip(
+                    h('.gray', iconBan()),
+                    'No QC flag was assigned. You have no permission to manage QC flag for this detector',
+                ),
```

Comment:
The reason not to put it directly in function is that according previous recommendation nothing should be rendered in case user has no permissions on `QcFlagsFor(...)OverviewPage`. In case `noPermissionContent` is not passed as argument there it will be needed to duplicate logic of `remoteDplDetectors.match(...)` from `createRunDetectorsAsyncQcActiveColumns`. Besides, I added observableData with detectors which for user has expert permissions